packages/overlays/stories/configuration.stories.mdx

```diff
@@ -0,0 +1,566 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+
+import './lion-demo-overlay.js';
+import './applyDemoOverlayStyles.js';
+
+<Meta title="Overlay System/Configuration" parameters={{ component: 'lion-demo-overlay' }} />
+
+# Overlay System - Configuration
+
+In total, we should end up with configuration options as depicted below, for all possible overlays.
+All boolean flags default to 'false'.
```

Comment:
This sentence originates from a design doc and now 'became truth'.

Maybe we can change it to something like:
```
The OverlayController has several configuration options. Via these options, we allow for all possible occurrences of overlays as described in "Overlay System Scope" (link).